docs/ops/arithmetic/Add_1.md

```diff
@@ -6,46 +6,48 @@
 
 **Short description**: *Add* performs element-wise addition operation with two given tensors applying multi-directional broadcast rules.
```

Comment:
`... applying broadcasting rule specified in the *auto_broacast* attribute.`

Reply:
Corrected.